src/components/Project/DnDColumnList.test.tsx

```diff
@@ -10,7 +11,7 @@ describe("<DnDColumnList />", () => {
     const secondColumnTitle = MOCKED_PROJECT.columns[1].name;
     const thirdColumnTitle = MOCKED_PROJECT.columns[2].name;
 
-    render(<DnDColumnList projectColumns={MOCKED_PROJECT.columns} />);
+    render(<DnDColumnList projectColumns={MOCKED_PROJECT.columns} />, { wrapper: TestRendererWithContext });
```

Comment:
Make sure to wrap component this custom function `TestRendererWithContext` to pass mocked `Project` to the component to test